nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh

```diff
@@ -41,16 +42,22 @@ magma_namespace=${magma_namespace:-orc8r}
 echo ""
 
 # Find NMS pod name
-nms_pod_name=$(kubectl -n $magma_namespace get pods --no-headers -o custom-columns=":metadata.name" | grep nms-magmalte)
+nms_pod_name=$(kubectl -n $magma_namespace get pods --no-headers -o custom-columns=":metadata.name" | grep nms-magmalte | head -n 1)
```

Comment:
Can we use this instead
NMS_POD=$(kubectl -n orc8r get pod -l  app.kubernetes.io/component=magmalte -o jsonpath='{.items[0].metadata.name}') ?